app/models/user.rb

```diff
@@ -2,4 +2,22 @@ class User < ActiveRecord::Base
   include Clearance::User
 
   has_many :itineraries
+  has_many :friendships
+  has_many :passive_friendships, :class_name => "Friendship", :foreign_key => "friend_id"
+  has_many :active_friends, -> { where(friendships: { approved: true}) }, :through => :friendships, :source => :friend
```

Comment:
Line is too long. [118/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.